docs/en/latest/design.md

```diff
@@ -59,7 +59,7 @@ Apache APISIX ingress for Kubernetes.
 
 ## CRD design
 
-Currently `apisix-ingress-controller` CRDs consist of 3 parts: ApisixRoute/ApisixUpstream. The design follows the following ideas.
+Currently `apisix-ingress-controller` CRDs consist of 2 parts: ApisixRoute/ApisixUpstream. The design follows the following ideas.
```

Comment:
```suggestion
Currently `apisix-ingress-controller` CRDs consist of 6 parts: ApisixRoute/ApisixUpstream/ApisixConsumer/ApisixTls/ApisixClusterConfig/ApisixPluginConfig. The design follows the following ideas.
```

This documentation is a bit outdated, but we can prioritize updating this section